tests/clients/test_pymysql.py

```diff
@@ -39,303 +37,314 @@ def setUp(self):
         END
         """
         setup_cursor = self.db.cursor()
-        for s in database_setup_query.split('|'):
-          setup_cursor.execute(s)
+        for s in database_setup_query.split("|"):
+            setup_cursor.execute(s)
 
         self.cursor = self.db.cursor()
-        self.recorder = tracer.recorder
+        self.recorder = tracer.span_processor
         self.recorder.clear_spans()
         tracer.cur_ctx = None
-
-    def tearDown(self):
+        yield
         if self.cursor and self.cursor.connection.open:
-          self.cursor.close()
+            self.cursor.close()
         if self.db and self.db.open:
-          self.db.close()
+            self.db.close()
         agent.options.allow_exit_as_root = False
 
-    def test_vanilla_query(self):
+    def test_vanilla_query(self) -> None:
         affected_rows = self.cursor.execute("""SELECT * from users""")
-        self.assertEqual(1, affected_rows)
+        assert 1 == affected_rows
```

Comment:
Always put the expected value on the right side of the assert statement. That way, it will be easier to change if necessary in the future and better to read.

Make this change all through this file, please.

```suggestion
        assert affected_rows == 1
```